src/styles/blocks/card.scss

```
@@ -0,0 +1,87 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__product-name {
+    margin-bottom: 4px;
+
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    border: 1px solid transparent;
+    background-color: $color-blue-accent;
+    border-radius: 5px;
+    transition: 0.3s;
```

It would be best if you explicitly named all properties that will change on transition
```suggestion
  transition: 0.3s property1 ease, 0.3s property2 ease;
```

---

Ok. Thx.